app/src/main/java/org/desarrolladorslp/teckersapp/service/AuthEndpoint.kt

```diff
@@ -0,0 +1,32 @@
+package org.desarrolladorslp.teckersapp.service
+
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class AuthEndpoint {
+
+    companion object {
+        private var authInstance: Retrofit? = null
+
+        fun instance(): Retrofit? {
+
+            if (authInstance == null) {
+
+                val client = OkHttpClient.Builder()
+                    .addInterceptor(AppCredentialsInterceptor())
+                    .build()
+
+                authInstance = Retrofit.Builder()
+                    .baseUrl("https://teckers-backend.herokuapp.com/")
```

Comment:
This URL must come from a configuration file

Reply:
Why did you mark as resolved this conversation? ... the URL is still hard coded

Reply:
because i fixed. I dont have the class AuthEndpoint. I have RetrofitManager.

Reply:
Look this commit **a2d5e6c**